schemas/oic.links.properties.core-schema.json

```diff
@@ -117,6 +117,17 @@
         "description": "Descriptive function of the Resource, contains a valid OCF enumeration value",
         "type": "string",
         "readOnly": true
+      },
+      "tag-locn": {
+        "allOf": [
```

Comment:
Should this definition include "readOnly": true?

Reply:
I'm confused by the comment.  Are you saying to remove the "readOnly" tag that exists (see line 128)?  Or something else.

Reply:
I'm confused now too. I didn't see that readOnly tag when I reviewed this the first time.  Looks good!